services/server/README.md

```diff
@@ -1,54 +1,135 @@
 # sourcify-server
 
-Sourcify's server for verifying contracts.
+Sourcify's server for verifying Solidity and Vyper smart contracts.
+
+The server uses [lib-sourcify](https://github.com/ethereum/sourcify/tree/main/packages/lib-sourcify) under the hood for contract verification logic. It provides REST API endpoints for users to submit new contracts for verification or retrieve verified contracts. The data is stored in a PostgreSQL database.
 
 ## Development
 
 ### Prerequisites
 
-- [Node.js](https://nodejs.org/en/) (v22)
-- (Optional) Postgres (See [Choosing the storage backend](#choosing-the-storage-backend))
-- (Optional) Docker and docker-compose for tests
+- [Node.js](https://nodejs.org/en/) (recommended v22)
+- Postgres 16 (or Docker)
 
 ## Quick Start
 
-1. Install
+First head to the project root directory and run the following commands:
+
+```bash
+cd ../..
+```
+
+### 1. Install dependencies
 
 ```bash
 npm install
 ```
 
-2. Change the server storage backend to a filesystem for easy start (API v2 won't be available). Create a `src/config/local.js`. See the [Config](#config) section below for details.
+### 2. Build the monorepo's packages
 
-```js
-const {
-  RWStorageIdentifiers,
-} = require("../server/services/storageServices/identifiers");
+```bash
+npx lerna run build
+```
 
-module.exports = {
-  storage: {
-    read: RWStorageIdentifiers.RepositoryV1,
-    writeOrWarn: [],
-    writeOrErr: [RWStorageIdentifiers.RepositoryV1],
-  },
-};
+### 3. Spin up a PostgreSQL database
+
+Go to the `services/database`
+
+```bash
+cd services/database
 ```
 
-3. Build the monorepo's packages
+Copy the `.env.template` file into a file named `.env`. Change values if they are different for your Postgres instance or use those defaults.
 
 ```bash
-npx lerna run build
+cp .env.template .env
+```
+
+Run Postgres with docker compose (note that this will not have a `postgres` root user. For production, you should run a Postgres instance and then add your own user).
```

Comment:
In the linked issue (https://github.com/ethereum/sourcify/issues/1983), the user specifically asked to give instructions of how to set up postgres, the database, the user and give read and write access to the user. Maybe we should give a few commands to explain how to do this.